client/signup/steps/website-content/index.tsx

```diff
@@ -39,13 +45,43 @@ function WebsiteContentStep( {
 	goToNextStep,
 	queryObject,
 }: WebsiteContentStepProps ) {
+	const [ formErrors, setFormErrors ] = useState< ValidationErrors >( {} );
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
 	const websiteContent = useSelector( getWebsiteContent );
 	const currentIndex = useSelector( getWebsiteContentDataCollectionIndex );
 	const siteId = useSelector( ( state ) => getSiteId( state, queryObject.siteSlug as string ) );
 	const siteCategory = useSelector( ( state ) => getDIFMLiteSiteCategory( state, siteId ) );
-	const dispatch = useDispatch();
-	const translate = useTranslate();
-	const [ formErrors, setFormErrors ] = useState< ValidationErrors >( {} );
+	const isWebsiteContentSubmitted = useSelector( ( state ) =>
+		isDIFMLiteWebsiteContentSubmitted( state, siteId )
+	);
+	const isSiteInformationLoaded = useSelector( ( state ) =>
+		isRequestingSite( state, siteId as number )
+	);
```

Comment:
`isRequestingSite` returns `true` if a network request is in progress. Should this be the following instead?
```
const isSiteInformationLoaded = useSelector( ( state ) =>
		! isRequestingSite( state, siteId as number )
);
```

Maybe we also need a placeholder while site data is being loaded?

This also causes some lag for a site for which the content has already been submitted:

https://user-images.githubusercontent.com/5436027/154262598-8ac759b9-a5e2-4a30-8c62-0f9a56d15d03.mp4

Reply:
I have returned null for now, since this is an edge case. We can probably add a placeholder in a later PR. If there is a simple placeholder that can be  easily added instead which would look better please do let me know. :)

Reply:
Maybe the `onboarding-placeholder` or `Spinner` component can work here? We can pick this up later, though :) https://github.com/Automattic/wp-calypso/blob/28b3c2a7056bd8088f2d1118f0a405eb58338c1a/packages/onboarding/styles/mixins.scss#L210